ru-intl/remap/ru/background.js

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2021 The Extra Keyboards for Chrome OS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+
+RuIntl cyrillic keyboard layout
+Based on the Russian standard keyboard layout
+The AltGr key to be used as the Compose key
+Updated 2024-09-05
+Version 1.2
+*/
+
+var contextID = 0;
+
+const keys = [
```

Comment:
Maybe a more specific name here? toprowkeys perhaps?

Reply:
In fact those keys are the non-letter keys for which the caps lock is deactivated. So, I named this variable "capsoffkeys"